humanlayer-wui/src/AppStore.ts

```diff
@@ -110,12 +110,29 @@ export const useStore = create<StoreState>((set, get) => ({
     })),
   refreshSessions: async () => {
```

Comment:
Note to re-check this, curious what the scenarios are where the client falls out of state with the server after this call